packages/core/src/config/snapwp-config-manager.ts

```diff
@@ -2,7 +2,7 @@
 import { isValidUrl, generateGraphqlUrl } from '@/utils';
 import { Logger } from '@/logger';
 import type { BlockDefinitions } from '@/props';
-
+import type { HTMLReactParserOptions } from 'html-react-parser';
```

Comment:
@SH4LIN / @justlevine How about a prettier rule to add newline after import statements?

Reply:
Make a 1-off PR against https://github.com/rtCamp/headless/issues/250

Though this might be covered by ESlint's (non-wp) Dependency groupings, ref https://github.com/rtCamp/snapwp/pull/61/files/24fd3d30122ada52b7836cdf5f13d1ae589843f2#r1958860096